### PR TITLE
Script to generate notebooks into GS locations

### PR DESCRIPTION
**Pull request by xbrianh**

### CODE REVIEW

**Review by juklucas — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -27,6 +27,12 @@ Notebooks are published with make commands, e.g.
 make publish/byod
 ```
 
+### ad-hoc publication
+A convenience script is provided to generate notebooks into arbitrary Google Storage locations.
+```
```
[the remark this answers is not in the document]
Comment (juklucas) [reply README.md:32]: Also, scripts is mispelled in the call example

[the remark this answers is not in the document]
Comment (xbrianh) [reply README.md:32]: gsutil is required throughout, which should be mentioned in the readme, and probably checked in common.mk

Will fix.

File: scripts/generate_to_gs.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+fi
+
+callysto_script=$1
+gs_dest=$2
+
+if [[ ! -f ${callysto_script} ]]; then
+    echo "${callysto_script}: No such file"
+    exit 1
+fi
+
```
[the remark this answers is not in the document]
Comment (xbrianh) [reply scripts/generate_to_gs.sh:22]: Seems reasonable. Will add.

**Pull request merged**